config/apiRoutes.js

```diff
@@ -6,4 +6,5 @@ module.exports = function(app) {
   // User routes
   app.post('/api/auth/signup', users.signup);
   app.post('/api/auth/login', users.login);
+  app.get('/api/auth/currentuser', users.currentUser);
```

Comment:
`currentuser` should be camelCase or snake_case preferably snake_case. Please change this.